scripts/for_final_metadata.R

```diff
@@ -0,0 +1,44 @@
+# Load necessary libraries
+library(GEOquery)
+library(dplyr)
+
+# Step 1: Read your original metadata file
+metadata <- read.csv("combined_data.csv", stringsAsFactors = FALSE)
```

Comment:
this is assuming that this file lives in a specific folder and will not be able to run. You need to set the main directory either as an input parameter or from a reference file that you read in